assembl/static2/js/app/components/debate/brightMirror/fictionPreview.jsx

```diff
@@ -100,9 +110,12 @@ const FictionPreview = ({
   const name = authorName || '';
   const author = `${ellipsis(name, FICTION_PREVIEW_NAME_MAX_CHAR)} - `;
 
+  const backgroundColor = color + (publicationState === PublicationStates.DRAFT ? FICTION_DRAFT_OPACITY : '');
```

Comment:
Same here. Is it possible to manage the opacity in the CSS?

Reply:
@paolina92 @amensouissi I managed the dynamic background color and opacity using RGBA. If I want to do it fully in CSS, I have to create a CSS class for each color in normal and each for the opacity as I couldn't find how to dynamically set opacity only to a previously set background color using just a class